var/spack/repos/builtin/packages/amdblis/package.py

```diff
@@ -15,8 +15,13 @@ class Amdblis(BlisBase):
     computationally intensive operations.
     """
 
+    _name = 'amdblis'
     homepage = "https://developer.amd.com/amd-aocl/blas-library/"
     url = "https://github.com/amd/blis/archive/2.1.tar.gz"
     git = "https://github.com/amd/blis.git"
 
     version('2.1', sha256='3b1d611d46f0f13b3c0917e27012e0f789b23dbefdddcf877b20327552d72fb3')
+
+    @property
+    def libs(self):
+        return find_libraries(['libblis'], root=self.prefix, recursive=True)
```

Comment:
If this is common with the non-patched `blis` package it should go into `blis` and be inherited here.

Reply:
Fixed.

Reply:
Thanks!